reactioncommerce/reaction
# Fix broken marketplace orders dashboard

Resolves #2758 

Marketplace orders dashboard was broken, caused by the order filtering PR that was merged that had some code missing. This restores things back to order (pun intended 🤓).

### How to test

- Clone this branch
- Have at least 2 orders
- Visit orders dashboard
- Test it out: select orders, set shipping statuses, capture payments
- Observe that everything works as before
